packages/tabs/addon/templates/components/nucleus-tabs.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="nucleus-tabs__list" role="tablist" aria-label={{description}}>
+  <div class="nucleus-tabs__list__container">
+    {{#each tabPanels key="id" as |tab index|}}
+      {{#nucleus-tabs/tab-list-item 
+          name=tab.name 
+          disabled=tab.disabled
+          controls=tab.id 
+          tabOrder=index 
+          currentSelected=currentSelected 
+          registerTabListItem=(action registerTabListItem) 
+          handleActivateTab=(action activateTab)}}
+        {{tab.name}}
+      {{/nucleus-tabs/tab-list-item}}
+    {{/each}}
+  </div>
+</div>
+
+{{yield (hash
+  panel=(component "nucleus-tabs/tab-panel")
+  props=(hash
```

Comment:
While I feel it's a little strong to be called anti-pattern (as some Ember devs would), I would say it's not the Ember convention to send props like this. Also, this implies that you have to watch for any changes in the `props` hash even to listen for one of the values which could have performance implications.

Reply:
Yes, makes sense. Changed as suggested.